instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlClientAttributesGetter.java

```diff
@@ -24,7 +24,12 @@ public interface SqlClientAttributesGetter<REQUEST>
   /**
    * Get the raw SQL statement. The value returned by this method is later sanitized by the {@link
    * SqlClientAttributesExtractor} before being set as span attribute.
+   *
+   * @deprecated Use {@link #getDbQueryText(Object)} instead.
    */
+  @Deprecated
   @Nullable
   String getRawStatement(REQUEST request);
+
+  String getDbQueryText(REQUEST request);
```

Comment:
should this be named instead?
```suggestion
  String getRawQueryText(REQUEST request);
```